packages/discussions/graphql/resolvers/_mutations/featureComment.js

```diff
@@ -0,0 +1,45 @@
+const { Roles } = require('@orbiting/backend-modules-auth')
+
+module.exports = async (_, args, context) => {
+  const {
+    id,
+    content
+  } = args
+  const {
+    pgdb,
+    user: me,
+    t,
+    loaders,
+    pubsub
+  } = context
+
+  Roles.ensureUserHasRole(me, 'editor')
+
+  const comment = await loaders.Comment.byId.load(id)
+  if (!comment) {
+    throw new Error(t('api/comment/404'))
+  }
+
+  const newComment = await pgdb.public.comments.updateAndGetOne(
+    { id },
+    {
+      featuredAt: new Date(),
+      featuredContent: content
```

Comment:
Currently it's possible to pass an empty string and then receive `null` for `featuredText` but a `featuredAt` date. This would currently create problems for the front teaser which always needs a `featuredText`.

Currently there is also noway of «unfeaturing». This might be needed because of accidental features or someone might change their mind.

My suggestion: Let's make `content` arg of the mutation nullable in the schema and null `featuredAt` and `featuredContent` when `content` is falsy. The alternative would be to ensure `content` is not empty and add an unfeature mutation.